src/main/java/org/lookout_studios/meals_management_system/meals_management_system/EstablishConnectionWithDatabase.java

```diff
@@ -0,0 +1,62 @@
+package org.lookout_studios.meals_management_system.meals_management_system;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+class accessData {
+    static String url = "jdbc:mysql://localhost:3306/mms_database";
+    static String username = "root";
+    static String password = "";
+    static String connectionInstanceName = "com.mysql.cj.jdbc.Driver";
+}
+class databaseData {
+    static String fridges = "select * from fridges";
+    static String productInstance = "select * from productinstance";
+    static String users = "select * from users";
+    static String MeasurementUnits = "select * from MeasurementUnits";
+}
+/**
+ * Class establishing connection with database.
+ * 
+ * @author Hubert Borysowski  
+ */
+public class EstablishConnectionWithDatabase {
+    /**
+     * Method establishing connection with database.
+     *
+     * <p><code>accessConnection = DriverManager.getConnection (accessData.url, accessData.username, accessData.password);</code> connects to the database using provided data.</p>
+     * <p><code>executeSqlStatement = accessConnection.createStatement ();</code> creates an SQL statement, so we can prepare the object to receive data.</p>
+     * <p><code>resultSet = executeSqlStatement.executeQuery (databaseData.fooTable);</code> receives and stores information about the current table.</p>
```

Comment:
You don't have to explain what the methods you're using do. They are documented by their authors. What matters is the documentation of your own code and what exactly it does.